src/bit.rs

```diff
@@ -179,9 +176,9 @@ pub(crate) struct BitReaderState {
 #[cfg(test)]
 mod tests {
     use super::*;
-    #[cfg(feature = "no_std")]
+    #[cfg(not(feature = "std"))]
     use core2::io;
-    #[cfg(not(feature = "no_std"))]
+    #[cfg(feature = "std")]
     use std::io;
```

Comment:
```suggestion
    use std::io;
    #[cfg(not(feature = "std"))]
    use alloc::vec::Vec;
```

Reply:
Feel free to remove the conditional compilation, if you want, but the `use alloc::vec::Vec` import is needed to build successfully.

Reply:
Done.